src/components/data/toolbar/DataDotMenu.js

```diff
@@ -135,6 +140,21 @@ function DataDotMenu(props) {
                                       />
                                   </MenuItem>,
                               ],
+                              selected && selected.length > 0 && (
+                                  <MenuItem
+                                      key={build(baseId, ids.ADD_TO_BAG_MI)}
+                                      id={build(baseId, ids.ADD_TO_BAG_MI)}
+                                      onClick={() => {
+                                          onClose();
+                                          onAddToBagSelected();
+                                      }}
+                                  >
+                                      <ListItemIcon>
+                                          <AddToBagIcon fontSize="small" />
+                                      </ListItemIcon>
+                                      <ListItemText primary={t("addToBag")} />
+                                  </MenuItem>
```

Comment:
There is also now the `RowDotMenu` which appears within each row in the data's table view.  You could also add this same menu item there.